public/js/views/management/pay-methods.jsx

```diff
@@ -42,7 +53,7 @@ export default class PayMethods extends Component {
           {this.renderChild()}
           <a className="button quiet add-pay-method" href="#"
             onClick={this.handleAddPayMethod}>{gettext('Add a new card')}</a>
-          <a className="button quiet delete" href="#"
+          <a className={deleteClasses} href="#"
```

Comment:
I see I confused that in my tests. I'll fix them up to use a relevant element in the attr tests. I'll also make the class test only for links.